### ISSUE TITLE
Ability to specify post permalink

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#76]
```
What new feature do you want?

Thanks for your chrome add scibefire. I liked it a lot. Will you consider 
adding a new feature about changing the permalink for the post? Thanks a lot.

Regards,
 Julia
```

-----
Original issue reported on code.google.com by `hope2...@gmail.com` on 23 Jun 2010 at 7:05

**Comment by GoogleCodeExporter** [comment]
-----
Original comment by `cfinke` on 1 Jul 2010 at 10:20
* Changed title: **Ability to specify post permalink**
* Changed state: **Accepted**
* Added labels: **Type-Enhancement**, **Priority-High**, **Milestone-1.1**

**Comment by GoogleCodeExporter** [comment]
```
Issue 87 has been merged into this issue.
```

-----
Original comment by `cfinke` on 2 Jul 2010 at 1:27

**Comment by GoogleCodeExporter** [comment]
```
Done in r66, will be included in version 1.1.
```

-----
Original comment by `cfinke` on 3 Jul 2010 at 11:05
* Changed state: **Fixed**
